### PR TITLE
docs(nx): added slack community link

### PR DESCRIPTION
**Pull request by Doginal**
Added a badge for the community slack channel.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,6 +178,10 @@ You are good to go!
 - [nx-examples](https://github.com/nrwl/nx-examples) repo has branches for different nx comments to display expected behavior and example app and libraries. Check out the branch (workspace, ngrx...) to see what gets created for you. More info on readme.
 - [xplat - Cross-platform tools for Nx workspaces](https://nstudio.io/xplat/)
 
+## Community Slack
+
+Are you looking for support while using @nrwl products? Why not ask the community! Join us on Slack [@nrwl/community](https://join.slack.com/t/nrwlcommunity/shared_invite/enQtNzMwMTQ2MjY0MDA1LWE4ZDZiYjM5ZjE0MWVjYmVmODM1MDEyZGU5NzA2YTM2MDM2ZDY3Yzg4YWU4MDI4YTcwZTM1ZWRjNzE5M2YzY2Y)!
```
Comment (vsavkin) [inline README.md]: Could you instead add it next to [Join the chat at https://gitter.im/nrwl-nx/community](https://gitter.im/nrwl-nx/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)?

So it's just another badge

**Comment by vsavkin** [comment]
@Doginal thank you for your PR! Could you change your commit message to match our format? (doc => docs)

File: README.md (hunk 1 above)
Comment (Doginal) [reply README.md]: Will do

**Comment by Doginal** [comment]
@vsavkin you are welcome. Updated the commit message to the proper format

**Comment by vsavkin** [comment]
Awesome. Thank you!

**Comment by brandonroberts** [comment]
@Doginal the Slack invite link has expired. Will you post a new one?

**Pull request merged**